deployments/examples/opencloud_full/.env

```diff
@@ -157,7 +157,7 @@ COMPANION_ONEDRIVE_SECRET=
 ## Default Enabled Services ##
 
 ### Apache Tika Content Analysis Toolkit ###
-# Tika (search) is enabled by default, comment if not required.
+# Tika (search) is disabled by default due to performance resasons
```

Comment:
reasons 😀

Reply:
and you deleted the dot.

Nitpicking 😄

Reply:
:((((((